### ISSUE TITLE
Crash more or less after one minute of loggin

### ISSUE THREAD

**Issue by Aoshureng** [Laskyyy/Create-Astral#189]
I'm in single player, I haven't added or removed any mods.

My game crashes after a more or less equal time (one minute).

I removed the belt at XYZ -3,19,-37 but it still crashes, are these the correct coordinates? And it is in XYZ on -3,19,-37? 
Thanks!

[crash-2023-07-08_09.23.59-server.txt](https://github.com/Laskyyy/Create-Astral/files/11990784/crash-2023-07-08_09.23.59-server.txt)

**Comment by Erdragh** [comment]
Ok so, yes that should be the correct coordinates. Did you remove the other blocks around that belt? The crash log seems to mention a funnel?

Also something completely different I noticed is that your RAM is probably not running at its max speed. Go into your bios and see whether you can enable AMD expo or XMP (not sure whether expo was already a thing for ryzen 5000 series)

**Comment by Aoshureng** [comment]
After several tests on my save (I didn't test on others), I found that the funnels in *(Any amount) mode on a moonstone farm (max speed drill) if a certain number of funnels is reach the game crashes. I know it started crashing roughly when I hooked up a storage bus to the crude storage unit (now that I remember). To solve the problem I put the funnel on top of a barrel that I extract with funnels at 64 items.

And thanks for the RAM info, it's now enabled.
![Crash](https://github.com/Laskyyy/Create-Astral/assets/105956791/bc906728-14db-430b-9cfe-02a0e1613695)

![No crash](https://github.com/Laskyyy/Create-Astral/assets/105956791/8661685b-b98d-4c89-af2b-55d7586a8fd4)

**Comment by Aoshureng** [comment]
I found how to fix the crash. To explain the setup, I farm moonstone with drills (more or less 16 drills) that I extract with chutes which are in turn extracted with funnels in "*" mode on belts that go into a crude storage unit which is connected with a storage bus (It started crashing around this time).
  To fix I connected chute-barrel-funnel(64 items)-belt-crude storage unit-storage bus. That way I don't crash anymore except that if I don't chunkload the area I will crash at some point.

Thanks for the RAM indeed it was not activated.
![Crash](https://github.com/Laskyyy/Create-Astral/assets/105956791/98d1e0f2-3745-41d0-a542-0b276500779d)
![No crash](https://github.com/Laskyyy/Create-Astral/assets/105956791/6a97d671-b9cb-484e-9934-f3d52c31fccb)
